src/geojson-types.js

```diff
@@ -0,0 +1,75 @@
+// @flow
```

Comment:
Did you write this? I think there is a package you can import.

Reply:
I looked at that, but nebula only supports a subset of GeoJSON (e.g. no `GeometryCollection`). Using our own types means we don't have to do `if (type !== 'GeometryCollection')` all over the place.

Reply:
ok, great.